src/middlewares/apiMiddleware/config.js

```diff
@@ -1,3 +1,3 @@
-export const EXPRESSIONS_URL = '/api/expressions';
-export const REGISTER_URL = '/user/register';
-export const LOGIN_URL = '/user/login';
+export const EXPRESSIONS_URL = '/expressions/';
+export const REGISTER_URL = '/user/register/';
```

Comment:
Is there any reason so that `/user` was removed from the Login URL, but not from the Register one?

Reply:
Not really.. In the API the `/login` is for the loginController, `/user` for the userController and they are 2 different controllers so that's why I modified the url also.